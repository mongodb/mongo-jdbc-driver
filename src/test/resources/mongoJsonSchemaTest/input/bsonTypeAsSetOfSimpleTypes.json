{
  "bsonType": "object",
  "properties": {
    "bar": {
      "bsonType": [
        "int",
        "string",
        "double"
      ]
    }
  }
}
