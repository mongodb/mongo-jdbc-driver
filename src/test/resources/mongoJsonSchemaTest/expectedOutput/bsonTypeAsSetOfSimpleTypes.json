{
  "bsonType": "object",
  "properties": {
    "bar": {
      "anyOf": [
        {"bsonType": "int"},
        {"bsonType": "string"},
        {
          "bsonType": "double"
        }
      ]
    }
  }
}
